src/rules/selector-max-specificity/README.md

```diff
@@ -0,0 +1,29 @@
+# selector-max-specificity
+
+Control the specificity of selectors.
+
```

Comment:
Add an example here (between the single line description and the expanded description) to make it clear that this rule applies to to each individual selector in a selector list. Similar to how https://github.com/stylelint/stylelint/tree/master/src/rules/selector-class-pattern#selector-class-pattern does it.